src/widgets/breadcrumb/__tests__/__snapshots__/breadcrumb-test.js.snap

```diff
@@ -1,47 +1,41 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`breadcrumb() render renders transformed items correctly 1`] = `
-<Breadcrumb
-  canRefine={true}
-  createURL={[Function]}
-  cssClasses={
+Object {
```

Comment:
are you aware the JSX didn't work here?

Reply:
See the section "Snapshot to Preact's `render` function have changed" in the original post.